lib/usd/ui/layerEditor/stageSelectorWidget.cpp

```diff
@@ -18,13 +18,16 @@
 #include "qtUtils.h"
 #include "stringResources.h"
 
+#include <pxr/pxr.h>
 #include <pxr/usd/usd/common.h>
 
 #include <QtCore/QSignalBlocker>
 #include <QtWidgets/QHBoxLayout>
 #include <QtWidgets/QLabel>
 
-Q_DECLARE_METATYPE(PXR_NS::UsdStageRefPtr);
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
I added the using directive and deleted all the explicit PXR_NS calls, seems to be the pattern we're following in most places.

Reply:
Huh, I thought internally we just talked about this and decided that we would stop using the using directive and always use PXR_NS.

Reply:
I typed in that comment and posted the review and THEN the internal message was sent out.  I think that sentence was what prompted it.

Reply:
I don't care either way.  I made the change after someone pointed out how in one of the files we had the using directive but then there was a mix of using PXR_NS or not and it wasn't consistent.  Here I just added it and removed because that's normally what I see in code, but if we want to go the other way then that's fine with me.  It does sound like we might make a sweep of the entire repo and get things consistent though, whatever we decide to do, so not sure if we necessarily need to go back and remove it all from this change.

Reply:
I would kind of prefer if we left this code alone for now. Once we make the decision about how we want to proceed via PXR_NS we can have a dedicated PR to fix that.

Reply:
So I'm suggesting you put back the PXR_NS to not confuse this PR with changes not related to the actual fix.